config/sync/user.role.os2loop_user_administrator.yml

```diff
@@ -144,6 +147,7 @@ permissions:
   - 'edit own os2loop_post content'
   - 'edit own os2loop_question content'
   - 'edit own os2loop_section_page content'
+  - 'edit terms in os2loop_media_library'
```

Comment:
This looks like stuff from https://github.com/itk-dev/os2loop/pull/161 and should be removed from here.